### ISSUE TITLE
"Get involved" modal stuck in front of vscode

### ISSUE THREAD

**Issue by stefaanv** [eclipse/paho.mqtt-spy#105]
How can I get rid of the Get Involved modal windows that is in front of my editor all the time ?

![image](https://user-images.githubusercontent.com/18347072/193402136-477cd09a-ab2d-4ff7-b9df-274247621ebe.png)

**Comment by dgoadby** [comment]
Yes, me too, As the mqtt-spy version has not changed I assume it is a problem caused by a java update I tool last week.

**Comment by sami79** [comment]
same here,

**Comment by yadayada** [comment]
Hitting the escape key worked for me.

**Comment by hydra** [comment]
I'm here to report that focusing the main window, and then pressing escape worked, attempting to press escape after clicking the nag window does nothing.

just casting my opinion here now, but this is an incredibly ill thought-out pop-up judging by the amount of comments here and the fact that we call had to come here at all!
